frontend/src/scenes/sessions/filters/EditFiltersPanel.tsx

```diff
@@ -173,8 +175,8 @@ export function EditFiltersPanel({ onSubmit }: Props): JSX.Element | null {
                                         }}
                                         data-attr="edit-session-filter"
                                     >
-                                        <strong>{item.label}</strong>
-                                        <DownOutlined style={{ fontSize: 12, color: 'rgba(0,0,0,0.5)' }} />
+                                        <PropertyKeyInfo value={item.label || 'Property'} />
```

Comment:
How come we're passing in 'Property' here? 😮

Reply:
Label is possibly undefined & needs a backup value... could also be empty string instead

Reply:
I believe this is a TS thing right? `label` can be `undefined`, not sure if it's better to pass an empty string? Or update the `PropertyKeyInfo` to expect undefined attributes?